manifest.json

```diff
@@ -8,7 +8,7 @@
     },
     "version": "1.0",
     "url": "https://github.com/YunoHost-Apps/my_webapp_ynh",
-    "license": "GPLv3",
+    "license": "free",
```

Comment:
Uh I don't know exactly what's the standard to put in this field, but is it really what we want to do ?

Reply:
It seems that licence must be: free or non-free, as descibed in the documentation
https://yunohost.org/#/packaging_apps_manifest_en

Reply:
But after checking the https://github.com/YunoHost/package_linter/blob/master/package_linter, other value can be accepted if they are part of https://spdx.org/licenses/
So other good value would be:

- GPL-3.0-only
- GPL-3.0-or-later

Reply:
yes there is no upstream app for this package but the maintainer added a GPL3 License in the package... 
So "free" may be the good value